relocations/quarkus-amazon-dynamodb-deployment/pom.xml

```diff
@@ -15,7 +15,7 @@
         <relocation>
             <groupId>io.quarkiverse.amazonservices</groupId>
             <artifactId>quarkus-amazon-dynamodb-deployment</artifactId>
-            <version>1.0.0</version>
+            <version>1.0.1</version>
             <message>${project.groupId}:${project.artifactId} was relocated to io.quarkiverse.amazonservices:${project.artifactId} and is now managed by the io.quarkus.platform:quarkus-amazon-services-bom:${project.version}. Please, update its groupId in the dependency declaration and import io.quarkus.platform:quarkus-amazon-services-bom:${project.version} in your project configuration. For more information about this change, please, refer to https://github.com/quarkusio/quarkus/wiki/Migration-Guide-2.6</message>
```

Comment:
Is the usage of `${project.version}` correct here?

Reply:
Yes, this message is specific because we use a BOM from the Platform so it's the Platform version (which is equal to the Core version as I always release both at the same time).